packages/ai-jsx/src/react/jit-ui/mdx.tsx

```diff
@@ -0,0 +1,115 @@
+import * as AI from '../core.js';
+import { ChatCompletion, SystemMessage } from '../../core/completion.js';
+import React from 'react';
+import { collectComponents } from '../completion.js';
+
+/**
+ * Use GPT-4 with this.
+ */
+export function MdxChatCompletion({ children, usageExamples }: { children: AI.Node; usageExamples: React.ReactNode }) {
```

Comment:
Not sure if this and `UICompletion` should be separate things (I see you were already wary of this).

Reply:
Yeah, I don't feel strongly, but at least for now, I think there's value in MDX as a standalone output format (e.g. an AI-driven static site generator).